lib/MessageDigest-browser.js

```diff
@@ -3,11 +3,7 @@
  */
 'use strict';
 
-require('setimmediate');
-
-const crypto = self.crypto || self.msCrypto;
-
-// TODO: synchronous version no longer supported in browser
+const crypto = require('isomorphic-webcrypto');
```

Comment:
How much does this bloat the package size? Is there a way for us to only load this for react-native? We had considerable size savings by just using native implementations before. Shouldn't react-native apps just pull this in on their own and expose it as a global?

Reply:
Always a good question (re package size). Far as I can tell: https://bundlephobia.com/result?p=isomorphic-webcrypto@2.3.8 (10.5k minified, 3.7k gzip'd). 

Pulling in crypto as a global, as a technique, is not a bad idea, worth exploring.

(Also, I very much wish that React Native had a similar section that the "browser" has in package.json. As far as I can tell, it just uses the "browser" section, actually. Need to investigate.)

Reply:
Ahhh you know what, RN straight up has a `"react native"` section of package.json, that acts exactly like the browser section.

Reply:
@dlongley should be fixed now.